ingest/ingest-people.py

```diff
@@ -250,12 +250,33 @@ def get_email(person):
         .one().value
 
 def get_website(person):
-    return Maybe.of(person).stream() \
+    weblinks = []
```

Comment:
that's correct. Previously it returned a string/literal
It's a new feature, so no prior dependencies.
I identified this should be an array because there are multiple websites in a vcard.
It's also a structure now because in the calling .html I saw there was no way to determine what type of weblink it was without adding the label.
I adjusted the new html ( which hasn't been deployed yet )